prisma/schema.prisma

```diff
@@ -10,19 +10,71 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum StationType {
+  Fixed
+  Mobile
+}
+
 model Station {
-  id   String @id @default(auto()) @map("_id") @db.ObjectId
-  name String
-  type String
-  sensors Sensor[]
-  deletedAt DateTime?
+  id            String   @id @default(auto()) @map("_id") @db.ObjectId
+  name          String
+  type          StationType
+  latitude      String
+  longitude     String
+  description   String
+  image_url     String
+  sensors       Sensor[]   // Define a relation to Sensor
```

Comment:
We should add this properties to track the changes by user :
- created_at
- created_by
- updated_at
- updated_by